src/shared/environment/Environment.php

```diff
@@ -125,7 +133,7 @@ private function disableXDebug(): void {
         \ini_set('xdebug.max_nesting_level', '8192');
         \ini_set('xdebug.show_exception_trace', 'off');
         // since `xdebug_disable` got removed in Xdebug 3 we have to check for its existance
-        if (function_exists('xdebug_disable')) {
+        if (\function_exists('xdebug_disable')) {
```

Comment:
php-cs-fixer did that. Didn't want to spend another commit just on that one line.